data/human/deep missions.txt

```diff
@@ -2431,3 +2431,320 @@ event "battle in arneb end"
+			`	You get the impression that while Thorleif is not ruined, this is an expensive and major loss for him. The moment the winds are calm, he demands Helm take you back to the mainland, and your ship. "This Captain," Thorleif's voice oozing with disgust, "is just as lucky as my so-called father. I want him gone."`
+			`	For most of the trip back to your ship Helm says nothing, he has been cool to you ever since you refused to help save the island. Finally, as you are preparing to disembark from his ship he stops you. "You've done Skaldgar no favors here. I'd appreciate it if we don't see from you again."`
+			`	Helm holds up his hand showing off the stone in his palm, "I took it while you were napping. Skaldgar made a mistake with you, and I will at least see the stone properly tended. Goodbye." Helm indicates towards the exit and waits for you to depart before cast off again in his ship.`
+				decline
+			label help
+			`	Thorleif nods and takes you and Helm outside into the storm, or Fimbulhrithr. Thorleif and Helm work the oars to steer the island away from the storm, while you are given the task of tying everything down that is liable to fly away.`
+			`	With the storm comes unnatural darkness, and the sky is filled with debris and fish and all manner of items that do not belong in it. You hurry as fast as you can, saving as much as you can.`
+			`	After hours of hard work, the storm begins to subside and the sea and sky begin to calm. An exhausted Thorleif and Helm stagger over to you, and Thorleif surveys his small island. "Without your help I would have lost many of the outbuildings. It was a lucky thing you came when you did."`
+			`	With that he turns and heads back to the main dwelling, with you and Helm in tired tow.`
+			choice
+				`	(Wash up and warm up in the main room around the fire.)`
+					goto fire
+			label fire
+			`	 You spend the evening warming yourself by a roaring fire telling Helm and Thorleif tales of your exploits in space, and many of the wonders you've seen.`
+			`	In turn they tell you about the sea beasts, leviathans, and krakens that they have met on the ocean currents of Norn. Of lost isles filled with forgotten treasure or strange people, or stories even, about the ancient elves who once aided their ancestors and are said to be around still.`
+			choice
+				`	(Ask about the elves.)`
+					goto elves
+				`	(Ask about lost treasure.)`
+					goto treasure
+			label elves
+			`	The idea of Elves, or Alfar as they call them, intrigues you and you ask what they know of them.`
+			`	"The Alfar," Thorleif begins, "are said to have been a great help to us in the early days. Some say they spared us from plagues and protected us from monsters. Many still offer sacrifices and prayers to them, so that they remember us and know that we are friends and allies still."`
```

Comment:
Are the Alfar supposed to be a pseudo-religious interpretation of the original Deep settlers' experience with the Pug? Seems to come off that way, curious if that's the intent.

Reply:
Yes.